docs/specification/current.md

```diff
@@ -159,7 +159,7 @@ curl
     "access_token": "some-opaque-fhir-access-token",
     "token_type": "Bearer",
     "expires_in": 300,
-    "scope": "patient/Patient.read patient/Observation.read",
+    "scope": "user/Patient.read user/Observation.read",
```

Comment:
Do we want to promote scopes that aren't constrained on a patient in context?  I think we can keep the patient-centric scopes.

Reply:
I think that the production functionality here varies.